README.md

```diff
@@ -18,6 +18,7 @@ Sunsnake is a Python inspired language for the web.
 * Indentation significant, no more {}-brackets :)
 * Significantly faster than Python.
 * Supports macros.
+* Supports controller input.
```

Comment:
This is not a language feature. It's a separate "module" you can use with Taptapir

Reply:
oh, i just assumed you wrote it too